### PR TITLE
Playing around with the new android sdk for twilio

### PR DESCRIPTION
**Pull request by DeTeam**
- [x] Upgrade Twilio Video SDK for Android
- [x] Upgrade Twilio Video SDK for iOS

### CODE REVIEW

File: android/src/main/java/com/actiotech/twiliovideorn/TwilioVideoModule.java [hunk 1]
```diff
@@ -510,7 +482,6 @@ public void disconnect() {
             }
             setAudioFocus(false);
             if (cameraCapturer != null) {
-                cameraCapturer.stopCapture();
```
Comment (DmitryTop) [inline android/src/main/java/com/actiotech/twiliovideorn/TwilioVideoModule.java:508]: Why `stopCapture` is removed?

File: android/src/main/java/com/actiotech/twiliovideorn/RNVideoViewGroup.java [hunk 2]
```diff
@@ -30,14 +29,14 @@ public RNVideoViewGroup(Context context) {
...
+                new tvi.webrtc.RendererCommon.RendererEvents() {
                     @Override
-                    public void onFirstFrame() {
+                    public void onFirstFrameRendered() {
 
                     }
 
                     @Override
-                    public void onFrameDimensionsChanged(int vw, int vh, int rotation) {
+                    public void onFrameResolutionChanged(int vh, int vw, int rotation) {
```
Comment (DmitryTop) [inline android/src/main/java/com/actiotech/twiliovideorn/RNVideoViewGroup.java]: Width and height order had changed

Comment (DeTeam) [reply android/src/main/java/com/actiotech/twiliovideorn/RNVideoViewGroup.java]: Fixed

**Review by DmitryTop — APPROVED** [review]

File: android/src/main/java/com/actiotech/twiliovideorn/TwilioVideoModule.java (hunk 1 above)
Comment (DeTeam) [reply android/src/main/java/com/actiotech/twiliovideorn/TwilioVideoModule.java:508]: Not sure, we need to test, but I have a feeling that it happens automatically now 🤔

**Comment by DeTeam** [comment]
Hm, tested a bit, still not happy with the implementation. Might require more in-depth checks.

**Comment by DeTeam** [comment]
Actually looked fine, will merge & then continue testing

**Pull request merged**